builder/dockerfile/parser/testfiles/escapes/Dockerfile

```diff
@@ -6,9 +6,7 @@ RUN apt-get \update && \
 ADD \conf\\" /.znc
 
 RUN foo \
-
 bar \
-
```

Comment:
Thanks @duglin. The comment has been removed and a new unit test has been added.